### ISSUE TITLE
v1.5.1  (VSC-1005)

### ISSUE THREAD

**Issue by JeonghunLee** [espressif/vscode-esp-idf-extension#838]
### OS

Windows

### Operating System version

Window 11

### ESP-IDF version

v4.2.3

### Python version

3.8.7

### Doctor command output

Sorry , 

### Description

ESP-IDF Monitor is not working properly in VS Code 
suddenly, it's not working after updated v1.5.1 

e.g.
set IDF_PATH=C:\Users\jhlee\esp\esp-idf
C:\Users\jhlee\.espressif\python_env\idf4.2_py3.8_env\Scripts\python.exe C:\Users\jhlee\esp\esp-idf\tools\idf_monitor.py -p COM5 -b 115200 --toolchain-prefix xtensa-esp32-elf-  **--target**  esp32 d:\Works\Project\mod610_sw\build\mxx610-app.elf

so I removed --target in dist/extension.js 

"--toolchain-prefix",this.config.toolchainPrefix];this.config.wsPort&&n.push("--ws","ws://localhost:"+this.config.wsPort),n.push(this.config.elfFilePath);

### Debug Message

```plain
Sorry, now not used OpenOCD based on VSCode
```

### I have checked existing issues, online documentation and the Troubleshooting Guide

- [X] I confirm I have checked existing issues, online documentation and Troubleshooting guide.

**Comment by brianignacio5** [comment]
Which command are you running ? Which device are you using ?

Could you provide a bit more information  about what is the error being shown ?

**Comment by JeonghunLee** [comment]
1. my device is esp32 
2. I used ESP-IDF monitor Button in VS Code 

it's not working properly because of --target argument 
```
PS D:\Works\Project\mod610_sw> set IDF_PATH=C:\Users\jhlee\esp\esp-idf
PS D:\Works\Project\mod610_sw> C:\Users\jhlee\.espressif\python_env\idf4.2_py3.8_env\Scripts\python.exe C:\Users\jhlee\esp\esp-idf\tools\idf_monitor.py -p COM5 -b 115200 --toolchain-prefix xtensa-esp32-elf- --target esp32 d:\Works\Project\mod610_sw\build\mxx610-app.elf
usage: idf_monitor - a serial output monitor for esp-idf [-h] [--port PORT] [--baud BAUD] [--make MAKE] [--encrypted]
                                                         [--toolchain-prefix TOOLCHAIN_PREFIX] [--eol {CR,LF,CRLF}]
                                                         [--print_filter PRINT_FILTER] [--decode-coredumps {info,disable}]      
                                                         elf_file
idf_monitor - a serial output monitor for esp-idf: error: argument elf_file: can't open 'esp32': [Errno 2] No such file or directory: 'esp32'
```

**Comment by JeonghunLee** [comment]
so resolved this issue by changing  C:\Users\jhlee\.vscode\extensions\espressif.esp-idf-extension-1.5.1\dist\extension.js
removed it below "n.push "--target" 
```
"--toolchain-prefix",this.config.toolchainPrefix];this.config.wsPort&&n.push("--ws","ws://localhost:"+this.config.wsPort),n.push(this.config.elfFilePath);
```

**Comment by brianignacio5** [comment]
The issue was that this argument should only be used for ESP-IDF 4.3 or higher. Making a fix in #850

**Comment by JeonghunLee** [comment]
I don't understand it, we are developing the products based on v4.2.3 now 

v4.2 ESP-IDF Maintenance period below 
    https://docs.espressif.com/projects/esp-idf/en/v4.4.3/esp32/versions.html#support-periods

Doesn't ESP-IDF v4.2.3 or Lower support anymore?

**Comment by brianignacio5** [comment]
Yes we do. Was an bug in the extension itself. Should be fixed with the PR #850

**Comment by JeonghunLee** [comment]
Ok I see , Thank you fast response
